controllers/webspherelibertyapplication_controller.go

```diff
@@ -612,6 +612,16 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 
 	instance.Status.Versions.Reconciled = lutils.OperandVersion
 
+	if instance.Spec.SemeruCloudCompiler != nil {
+		message := "Start Semeru Compiler reconcile"
+		reqLogger.Info(message)
+		err, message = r.reconcileSemeruCompiler(instance)
+		if err != nil {
+			reqLogger.Error(err, message)
+			return r.ManageError(err, common.StatusConditionTypeReconciled, instance)
+		}
+	}
```

Comment:
if semeruCloudCompiler is nil, the JITServer resources (Deployment and Service) should be removed